value.go

```diff
@@ -1,6 +1,10 @@
 package duckdb
 
 import (
+	"fmt"
+	"reflect"
+
+	duckdb_go_bindings "github.com/duckdb/duckdb-go-bindings/darwin-arm64"
```

Comment:
Remove this import and all usages - we must always go through the `mapping` package.